src/api/Config/types.ts

```diff
@@ -53,6 +53,7 @@ export interface Configuration {
   entityResolution?: EntityResolutionConfig;
   entityResolutionSourceKey?: string;
   cluster?: ClusterConfig;
+  troubleshooting: TroubleshootingConfig;
```

Comment:
@EricMallard this property is actually only set for admin users, so it's optional. I'm going to push a fix

```suggestion
  troubleshooting?: TroubleshootingConfig;
```